guides/schematics.md

```diff
@@ -0,0 +1,53 @@
+Angular Material comes packaged with Angular CLI schematics to make
+creating new components easy.
+
+## Install Schematics
+Schematics come packaged with Angular Material, so once you have
+installed the npm package, they will be available via the Angular CLI.
+
+## Packaged Schematics
+Angular Material has 4 schematics it comes packaged with:
+
+- Material Shell
+- Navigation
+- Dashboard
+- Table
+
+### Shell Schematic
+The shell schematic will help you quickly add Material to a new project. 
+This schematic will:
+
+- Ensure project depedencies in `package.json`
+- Ensure project depedencies in your app module
+- Add Prebuilt or Setup Custom Theme
+- Add Roboto fonts to your index.html
+- Apply simple CSS reset to body
+
+```
+ng add @angular/material
+```
+
+### Navigation Schematic
+The navigation schematic will create a new component that includes
+a toolbar with the app name and the side nav pre-configured to be
+responsive.
+
+```
+ng g @angular/material:material-nav
```

Comment:
I think it is clearer if you don't use the `g` alias, and instead have:

`ng generate @angular/material:material-nav`